packages/integration-tests/Collections.js

```diff
@@ -18,6 +18,7 @@ const buildCollection = (overrides = {}) => ({
   granuleId: '^[^.]+$',
   granuleIdExtraction: '^([^.]+)\..+$',
   sampleFileName: 'asdf.jpg',
+  url_path: randomId('url-path-'),
```

Comment:
What prompted this change?

Reply:
I ran into a case where either the staging file or the final ingested file (I forget) failed to be cleaned up and, the next time the test ran, it failed because it tried to put a file where there was already an existing file. This randomizes where the file is written to.